Recipes/Recipes/Recipe Detail Screen/Cells/ImageTableViewCell.swift

```diff
@@ -0,0 +1,149 @@
+    private let timerLabel = UILabel()
+
+    // MARK: - Public Methods
+
+    func configureCell(title: String, image: String, weight: Int, cookingTime: Int) {
+        backgroundColor = .white
+        self.title = title
+        self.image = image
+        self.weight = weight
+        self.cookingTime = cookingTime
+        addViews()
+        setupUI()
+        setConstraints()
+    }
+
+    // MARK: - Private Methods
+
+    private func addViews() {
+        contentView.addSubview(recipeTitleLabel)
+        contentView.addSubview(recipeImageView)
+        
+        recipeImageView.addSubview(roundView)
+        roundView.addSubview(potImageView)
+        roundView.addSubview(weightLabel)
+        recipeImageView.addSubview(timerBackgroundView)
+        timerBackgroundView.addSubview(timerImageView)
+        timerBackgroundView.addSubview(timerLabel)
+    }
+
+    private func setupUI() {
+        recipeTitleLabel.text = title
+        recipeTitleLabel.font = UIFont.verdanaBold20()
+        recipeTitleLabel.textAlignment = .center
+
+        recipeImageView.image = UIImage(named: image)
+        recipeImageView.layer.cornerRadius = 24
+        recipeImageView.contentMode = .scaleAspectFit
+
+        roundView.backgroundColor = UIColor.background01().withAlphaComponent(0.6)
+        roundView.layer.cornerRadius = 24
+        
+        potImageView.image = UIImage(named: Constants.potIcon)
+        potImageView.tintColor = .white
+        
+        weightLabel.text = "\(weight) g"
+        weightLabel.textColor = .white
+        weightLabel.font = UIFont.verdana10()
+        weightLabel.textAlignment = .center
+        
+        timerBackgroundView.backgroundColor = UIColor.background01().withAlphaComponent(0.6)
+        timerBackgroundView.layer.maskedCorners = [.layerMaxXMaxYCorner, .layerMinXMaxYCorner, .layerMinXMinYCorner]
+        timerBackgroundView.layer.cornerRadius = 24
+        
+        timerImageView.image = UIImage(named: Constants.timerIcon)
+        timerImageView.tintColor = .white
+        
+        timerLabel.numberOfLines = 2
+        timerLabel.text = Constants.cookingTime + String(cookingTime) + Constants.min
+        timerLabel.textColor = .white
+        timerLabel.font = UIFont.verdana10()
+        timerLabel.textAlignment = .center
+    }
+
+    private func setConstraints() {
+        recipeTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        recipeImageView.translatesAutoresizingMaskIntoConstraints = false
+        roundView.translatesAutoresizingMaskIntoConstraints = false
+        potImageView.translatesAutoresizingMaskIntoConstraints = false
+        weightLabel.translatesAutoresizingMaskIntoConstraints = false
+        timerBackgroundView.translatesAutoresizingMaskIntoConstraints = false
+        timerImageView.translatesAutoresizingMaskIntoConstraints = false
+        timerLabel.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
```

Comment:
```suggestion
        NSLayoutConstraint.activate([
```
коснтрейнты мы уже давно разносим по методам глянь как у других